libraries/griptape_nodes_library/workflows/templates/compare_prompts.py

```diff
@@ -53,36 +55,38 @@
 """
 top_level_unique_values_dict = {
-    "625fb631-4e7d-46de-beb3-377daaf2f54f": pickle.loads(
+    "59c9d7e2-f62c-4e31-989e-9ad1f9ae3dae": pickle.loads(
         b'\x80\x04\x95\xbf\x01\x00\x00\x00\x00\x00\x00X\xb8\x01\x00\x00This workflow serves as the lesson material for the tutorial located at:\n\nhttps://docs.griptapenodes.com/en/stable/ftue/03_compare_prompts/FTUE_03_compare_prompts/\n\nThe concepts covered are:\n\n- How to use one TextInput node to feed to multiple other inputs\n- Different approaches to prompt engineering\n- The GenerateImage "Enhance Prompt" feature and how it works behind the scenes\n- Comparing the results of different prompting techniques\n\x94.'
     ),
-    "8f23b4aa-f0df-4677-bdea-b580fca4b0d9": pickle.loads(
+    "7d8312d9-2388-4067-8af4-b68ab07145e2": pickle.loads(
         b"\x80\x04\x95\xf9\x00\x00\x00\x00\x00\x00\x00\x8c\xf5If you're following along with our Getting Started tutorials, check out the next suggested template: Photography_Team.\n\nLoad the next tutorial page here:\nhttps://docs.griptapenodes.com/en/stable/ftue/04_photography_team/FTUE_04_photography_team/\x94."
     ),
-    "83428258-61af-4336-8b55-135a34cf660c": pickle.loads(
+    "b413aa82-8153-483e-be20-14f3368fcea8": pickle.loads(
         b"\x80\x04\x95\xcf\x01\x00\x00\x00\x00\x00\x00X\xc8\x01\x00\x00Enhance the following prompt for an image generation engine. Return only the image generation prompt.\nInclude unique details that make the subject stand out.\nSpecify a specific depth of field, and time of day.\nUse dust in the air to create a sense of depth.\nUse a slight vignetting on the edges of the image.\nUse a color palette that is complementary to the subject.\nFocus on qualities that will make this the most professional looking photo in the world.\n\x94."
     ),
-    "5ed879bb-20cf-4479-815d-c9915fbd262b": pickle.loads(
+    "cedef26f-a10b-4cc4-87b2-8fce87b1b0b0": pickle.loads(
         b"\x80\x04\x95\x15\x00\x00\x00\x00\x00\x00\x00\x8c\x11A happy capybara.\x94."
     ),
-    "8fcd5bef-f968-40c7-ae47-f48517885ad0": pickle.loads(
+    "b5d083df-06bb-4077-b674-dce735d728bf": pickle.loads(
         b"\x80\x04\x95\x08\x00\x00\x00\x00\x00\x00\x00\x8c\x04\\n\\n\x94."
     ),
-    "844bc3fe-3f08-4d8a-bd48-147d2899a8c5": pickle.loads(b"\x80\x04\x89."),
-    "a0a5d21b-2a86-420f-b337-63577863f06e": pickle.loads(
+    "804f9190-0e16-4921-9870-ae97fb528029": pickle.loads(b"\x80\x04\x89."),
+    "4b05c025-83bc-4d9e-9f73-95f1106ec4bd": pickle.loads(
         b"\x80\x04\x95\xf9\x00\x00\x00\x00\x00\x00\x00}\x94(\x8c\x04type\x94\x8c\rImageArtifact\x94\x8c\x02id\x94\x8c a1d85e8dfa5745b7a39be55cca4660fb\x94\x8c\treference\x94N\x8c\x04meta\x94}\x94(\x8c\x05model\x94\x8c\x08dall-e-3\x94\x8c\x06prompt\x94\x8c\x1fA capybara eating with utensils\x94u\x8c\x04name\x94\x8c$image_artifact_250411205314_ll63.png\x94\x8c\x05value\x94\x8c\x00\x94\x8c\x06format\x94\x8c\x03png\x94\x8c\x05width\x94M\x00\x04\x8c\x06height\x94M\x00\x04u."
```

Comment:
I think we want the default for this one to be gpt-image-1 so we get better results.. just a suggestion